lib/tasks/generateCachebusterInfo.js

```diff
@@ -10,17 +37,20 @@ const resourceFactory = require("@ui5/fs").resourceFactory;
  * @param {module:@ui5/fs.AbstractReader} parameters.dependencies Reader or Collection to read dependency files
  * @param {Object} parameters.options Options
  * @param {string} parameters.options.namespace Namespace of the application
+ * @param {('time'|'hash')} [parameters.options.signatureType='time'] Type of signature to be used
```

Comment:
IMO the type {('time'|'hash')} needs to be {string} and the options should be in the text - with the optional value brackets and the default value I am fine - but if @matz3 or @RandomByte is  fine with the syntax I am OK with it

Reply:
Yeah, I was wondering the same. Currently looks like this (`npm run jsdoc`):

![signatureTypeJSDoc](https://user-images.githubusercontent.com/1589939/56690219-3c7ad800-66dd-11e9-8493-6088df4799bb.png)

I'm with Peter that the type should still be string.

Reply:
Got it, back to `{string}` then 👍